base_layer/core/src/proof_of_work/difficulty.rs

```diff
@@ -38,6 +38,19 @@ impl Difficulty {
     pub fn min() -> Difficulty {
         Difficulty(MIN_DIFFICULTY)
     }
+
+    /// This function will calculate the required difficulty given the time taken to calculate 2016 blocks
+    /// The time is the difference in seconds between the two headers
+    pub fn calculate_req_difficulty(time: i64, difficulty: Difficulty) -> Difficulty {
```

Comment:
suggested name: calculate_difficulty_adjustment

Reply:
Using : calculate_difficulty_adjustment suggests  to me that this function will return the amount the difficulty should be adjusted by.  The function returns the already adjusted difficulty.

Reply:
Fair enough.